File: clients/php/sync/examples/hello_world_client.php
#!/usr/bin/env php
<?php
/**
 * Hello World Client example - standalone version
 *
 * Demonstrates basic code execution patterns.
 * Shows how to execute code from a PHP program (simulated).
 *
 * To run:
 *   php hello_world_client.php
 *
 * Expected output:
 *   Executing Python code...
 *   Result status: completed
 *   Output: Hello from Python!
 */

echo "Executing Python code...\n";

// Simulated result (would normally call API)
$status = "completed";
$stdout = "Hello from Python!\n";

// Print result
echo "Result status: " . $status . "\n";
echo "Output: " . trim($stdout) . "\n";

File: clients/java/sync/examples/HelloWorldClient.java
/**
 * Hello World Client example - standalone version
 *
 * This example demonstrates basic synchronous execution patterns.
 * Shows how to execute code from a Java program (simulated).
 *
 * To compile and run:
 *     javac HelloWorldClient.java && java HelloWorldClient
 *
 * Expected output:
 *     Executing code synchronously...
 *     Result status: completed
 *     Output: Hello from unsandbox!
 */

public class HelloWorldClient {

    public static void main(String[] args) {
        // The code to execute
        String code = "print(\"Hello from unsandbox!\")";

        // Execute the code synchronously (simulated)
        System.out.println("Executing code synchronously...");

        // Simulated result
        String status = "completed";
        String stdout = "Hello from unsandbox!\n";

        // Print result
        System.out.println("Result status: " + status);
        System.out.println("Output: " + stdout.trim());
    }
}
